packages/core/src/api/resolvers/entity/order-line-entity.resolver.ts

```diff
@@ -32,6 +34,18 @@ export class OrderLineEntityResolver {
         @Parent() orderLine: OrderLine,
     ): Promise<Asset | undefined> {
         if (orderLine.featuredAsset !== undefined) {
```

Comment:
I believe the tests are failing because you are doing an exact `!==` check for `undefined`. So if `orderLine.featuredAsset` is `null`, then the "if" statement will be true, and the program will go to line 40 and attempt to access `featuredAsset.preview`, which results in the error you see in the e2e tests: 

```
Error: Cannot read properties of null (reading 'preview')
```

So the solution should be to use `!=`, which will check for _both_ `null` and `undefined`.